### PR TITLE
Add week notes to What's new and make changes after Style Council

### PR DESCRIPTION
**Pull request by sarawilcox**
## Description
Add week notes to blog post section.
Make a few changes to content style guide and images component after last Style Council meeting

## Checklist

- [ ] Tested against the [NHS.UK frontend testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [x] Code follows the [NHS.UK frontend coding standards](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/coding-standards.md)
- [ ] Version number has been increased in `package.json` (using [SEMVER](https://semver.org/))
- [ ] CHANGELOG entry - not yet
- [ ] [Whats new page](https://service-manual.nhs.uk/whats-new) entry - yes, included here
- [ ] Page updated date - not yet - I need to do this next.

### CODE REVIEW

**Comment by sarawilcox** [comment]
Hi Karl, would you mind reviewing this and adding the image for Catherine's week notes, please?

I'll then do the changelog and update the page dates, which I forgot to do! 

Thanks a lot.

**Comment by chrimesdev** [comment]
Do we still need the `blog-post_feature-nhsd.jpg` file now that we have a new featured image or can this file be deleted? 

Not a massive issue however we can reduce the `blog-post_feature-catherine.jpg` file size by 25% using a image size compression website such as https://tinypng.com - the image goes from `175.9KB` to `131.3 KB`.

File: app/views/whats-new/updates.njk [hunk 1]
```diff
@@ -10,7 +10,39 @@
 
 {% block bodyContent %}
 
-<h2>July 2021</h3>
+<h2>August 2021</h2>
+
+  <table class="nhsuk-table">
+    <caption class="nhsuk-table__caption nhsuk-u-visually-hidden">Updates to the service manual in July 2021</caption>
```
Comment (chrimesdev) [inline app/views/whats-new/updates.njk:16]: Should the end of this setence be: August 2021

File: app/views/whats-new/blog-posts.njk [hunk 2]
```diff
@@ -10,6 +10,14 @@
 
 {% block bodyContent %}
   <h2><span class="nhsuk-u-visually-hidden">Blog posts from </span>2021</h2>
+<ul class="nhsuk-list nhsuk-list--border nhsuk-u-margin-bottom-5">
```
Comment (chrimesdev) [inline app/views/whats-new/blog-posts.njk:13]: Minor thing but can this line be indented to match the end tag

**Comment by sarawilcox** [comment]
> Do we still need the `blog-post_feature-nhsd.jpg` file now that we have a new featured image or can this file be deleted?
> 
> Not a massive issue however we can reduce the `blog-post_feature-catherine.jpg` file size by 25% using a image size compression website such as https://tinypng.com - the image goes from `175.9KB` to `131.3 KB`.

Yes, we can get rid of old images. Shall we aim to reduce image size in future?

**Comment by sarawilcox** [comment]
Thanks for spotting the date. I'll sort that out and the <ul> tomorrow when I review the whole thing and prep the changelog.

**Comment by karlgoldstraw** [comment]
> > Do we still need the `blog-post_feature-nhsd.jpg` file now that we have a new featured image or can this file be deleted?
> > Not a massive issue however we can reduce the `blog-post_feature-catherine.jpg` file size by 25% using a image size compression website such as https://tinypng.com - the image goes from `175.9KB` to `131.3 KB`.
> 
> Yes, we can get rid of old images. Shall we aim to reduce image size in future?

I already optimised the image for web when I exported it, but I suppose further reduction could happen. 

Another thought I had about this, was that the 'full size' width of the featured image is currently 1920px. Is there ever a situation where such a large width is needed? 

For example, before the image breaks into two columns, the single stacked column is only ever 768px MAX not even deducting padding etc. So I don't think there's a need for a width over anything like 768px? @chrimesdev what do you think?

**Comment by chrimesdev** [comment]
@karlgoldstraw yeah if the image max size is only ever going to be `768px` on stacked then `1920px` may be a bit excessive - worth trying to see if theres any loss in quality

**Comment by chrimesdev** [comment]
I think the `app/assets/blog-post_feature-nhsd.jpg` file can be deleted all together @karlgoldstraw, its not being used anywhere now there is a new featured blog post

**Comment by karlgoldstraw** [comment]
I reduced the image widths to 768px (kept the same ratio) and theres very little difference visually. The file sizes have reduced from the 171kb range to the 40kb range. Win win!

File: app/views/whats-new/updates.njk (hunk 1 above)
Comment (karlgoldstraw) [reply app/views/whats-new/updates.njk]: I've done this

File: app/views/whats-new/blog-posts.njk (hunk 2 above)
Comment (karlgoldstraw) [reply app/views/whats-new/blog-posts.njk]: I've done this

**Comment by chrimesdev** [comment]
Just needs a `CHANGELOG` entry and then this can be merged

**Comment by sarawilcox** [comment]
And I need to fix a couple of page dates. Will do tomorrow morning.

**Comment by sarawilcox** [comment]
Back to you to approve @karlgoldstraw

**Review by karlgoldstraw — APPROVED** [review]
All looks good! 👍

**Pull request merged**